sourcegraph/sourcegraph
# Add development mode icons for browser extension

closes https://github.com/sourcegraph/sourcegraph/issues/33587

Changes browser extension icons in dev mode.

## Test plan

- Run extension `yarn --cwd client/browser run dev:chrome`
- Check extension icon
- Set extension to inactive and check icon again


## App preview:

- [Web](https://sg-web-naman-browser-extension-dev-icons.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-qnalznnuck.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
